lua/net.lua

```diff
@@ -1,10 +1,14 @@
 local gatekeeperc = require("gatekeeperc")
+local ffi = require("ffi")
 
 local M = {}
 
 -- Function that sets up the network.
 function M.setup_block()
 
+	local front_ports = {"0000:83:00.0"}
+	local back_ports = {"0000:85:00.0"}
```

Comment:
We need to use the name that the Linux kernel uses to name network interfaces. Using PCI addresses would be a big surprise to network administrators in a negative way. A pointer to start is to look at the code of command `xip`. `xip` has to interpret interface names for some cases. It should be possible to map interface names to PCI addresses.